src/main/java/org/broadinstitute/consent/http/models/DataAccessRequest.java

```diff
@@ -42,6 +46,8 @@
 
   @JsonProperty public Timestamp updateDate;
 
+  @JsonProperty public ArrayList<String> countCollaborators = new ArrayList<>();
+
```

Comment:
I don't think we need this method. You can get the collaborators by looking at the `DataAccessRequestData` object instead.